cpp/CMakeLists.txt

```
@@ -277,7 +277,7 @@ rapids_cpm_init()
 # Not using rapids-cmake since we never want to find, always download.
 CPMAddPackage(
   NAME rapids_logger GITHUB_REPOSITORY rapidsai/rapids-logger GIT_SHALLOW TRUE GIT_TAG
-  14bb233d2420f7187a690f0bb528ec0420c70d48
+  f9b007ae30f924de2dca8c585334f0d55331d0b8 VERSION f9b007ae30f924de2dca8c585334f0d55331d0b8
```

Temporarily changing this to verify that things work as expected.
```suggestion
  fix/include_dir VERSION f9b007ae30f924de2dca8c585334f0d55331d0b8
```